cmd/wshd/main.go

```diff
@@ -393,13 +432,40 @@ func main() {
 			return
 		}
 
+		encryption := wampshell.NewEncryptionManager(sess)
+		if err = encryption.Setup(); err != nil {
+			log.Fatal(err)
+		}
+
+		fmt.Println(sess.Details().Realm())
```

Comment:
```suggestion
```

Reply:
Thanks! The changes have already been tested, and as we discussed, the realm option will be handled in a separate PR, so we don’t need to include it here.